uiowa/uiowa
# Refactor Drush cache usage and invalidate Twig cache on deployment.

Resolves #1748 

Also aims to solve Drush cache collisions with cron and hopefully allow for simultaneous deployments. Currently in dev.

### Testing
- Run `blt uiowa:multisite:cron` locally and see that it works.
- See that cron is running for sites on dev every 3 hours staggered. Documented in the wiki.
- Not much beyond that, unfortunately.
